src/lib/com/yetanalytics/xapipe/filter.clj

```diff
@@ -26,7 +26,9 @@
 (defn get-profile
   "Get a profile from the specified URL or throw"
   [url]
-  (try (per-json/json->edn (slurp url) :keywordize? true)
+  (try (json/parse-string-strict (slurp url) #(if (= % "@context")
+                                                :_context
+                                                (keyword nil %)))
```

Comment:
As I mentioned before Persephone now provides `coerce-profile` and `coerce-statement` in its `util.json` namespace.

Reply:
Yep, those are good to provide, but in this case it would be a little extra work. By providing awareness in the `key-fn` passed to cheshire we have the same effect